lib/csv.rb

```diff
@@ -547,6 +547,21 @@ def to_csv(**options)
     end
     alias_method :to_s, :to_csv
 
+    #
+    # Extracts the nested value specified by the sequence of name objects by calling dig at each step,
```

Comment:
I don't think that `the sequence of name objects` is good description.
`the +index+ or +header+ object and the followed sequence of +index+ objects` (this English may be broken) or something will be better.